src/util/piped_process.cpp

```diff
@@ -138,9 +138,9 @@ piped_processt::piped_processt(const std::vector<std::string> &commandvec)
   // Use process ID as a unique ID for this process at this time.
   base_name.append(std::to_string(GetCurrentProcessId()));
   const std::string in_name = base_name + "\\IN";
-  child_std_IN_Rd = CreateNamedPipe(
+  child_std_IN_Wr = CreateNamedPipe(
```

Comment:
Random place for comment: there are a bunch of typos in the commit message.

Reply:
I have now corrected a bunch of typos in the commit message.